From 65981138396f62b0c9b6114e7055fa87f1a7b129 Mon Sep 17 00:00:00 2001
From: Michael Achenbach <machenbach@chromium.org>
Date: Fri, 30 Jun 2023 13:33:14 +0000
Subject: [PATCH] Revert "[fuchsia] Re-enable -Wctad-maybe-unsupported"

This reverts commit 02f283f4aa8a3cfef60324969f9f20a4535ac79d.

Reason for revert:
The fix is not yet in V8's deps'ed fuchsia sdk and this
breaks the bots:
https://ci.chromium.org/ui/p/v8/builders/ci/V8%20Fuchsia%20-%20builder/21745/overview

Original change's description:
> [fuchsia] Re-enable -Wctad-maybe-unsupported
>
> fxbug.dev/127411 has been fixed.
>
> Change-Id: If12f13bff93e652202d26f6f0c939bc212e77feb
> Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4641165
> Commit-Queue: Yifei Teng <yifeit@google.com>
> Reviewed-by: Patrick Thier <pthier@chromium.org>
> Cr-Commit-Position: refs/heads/main@{#88573}

Change-Id: I3f53fcda08d0ae96fc24dff95fb3e1034640671d
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4660604
Auto-Submit: Michael Achenbach <machenbach@chromium.org>
Commit-Queue: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Bot-Commit: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#88592}
---
 BUILD.gn | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/BUILD.gn b/BUILD.gn
index 0320d1d7e00..b6968cb18af 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -1437,10 +1437,13 @@ config("toolchain") {
       "-Wno-shadow",
     ]
 
-    # Google3 enables this warning, so we should also enable it to find issue
-    # earlier. See https://reviews.llvm.org/D56731 for details about this
-    # warning.
-    cflags += [ "-Wctad-maybe-unsupported" ]
+    # TODO(fuchsia:127411): Re-enable once FIDL bindings are compatible.
+    if (!is_fuchsia) {
+      # Google3 enables this warning, so we should also enable it to find issue
+      # earlier. See https://reviews.llvm.org/D56731 for details about this
+      # warning.
+      cflags += [ "-Wctad-maybe-unsupported" ]
+    }
 
     if (v8_current_cpu == "x64" || v8_current_cpu == "arm64" ||
         v8_current_cpu == "mips64el" || v8_current_cpu == "riscv64") {
-- 
2.35.1

